### PR TITLE
feat(circleci): update spec

### PR DESCRIPTION
**Pull request by jerdog**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Update the `circleci.ts` CLI file with more info and new commands

**What is the current behavior? (You can also link to an open issue here)**
Using older circleci CLI commands

**What is the new behavior (if this is a feature change)?**
Uses updated circleci CLI commands

**Additional info:**

### CODE REVIEW

**Comment by jerdog** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by fedeci** [comment]
@jerdog please ping me when this is ready!

**Comment by jerdog** [comment]
Will do @fedeci - ran into a wall with timing this weekend but will work on it this week

**Comment by fedeci** [comment]
Don't worry man, no pressure!

**Comment by jerdog** [comment]
@fedeci this should be good to go now!

File: src/circleci.ts [hunk 1]
```diff
@@ -131,30 +368,333 @@ const completionSpec: Fig.Spec = {
...
+          options: [
+            {
+              name: "--context string",
+              description: "Policy context for decision, default is 'config'",
+            },
+            {
+              name: "--enabled",
+              description:
+                "Enable/disable policy decision evaluation in build pipeline",
+              args: [
```
Comment (grant0417) [inline src/circleci.ts:684]: These would be better with this: https://fig.io/docs/reference/option#requiresequals

File: src/circleci.ts [hunk 2]
```diff
@@ -131,30 +368,333 @@ const completionSpec: Fig.Spec = {
...
+            },
+          ],
+        },
+        {
+          name: "settings",
+          description:
+            "Get/set policy decision settings (To read settings: run command without any settings flags)",
+          options: [
+            {
+              name: "--context string",
```
Comment (fedeci) [inline src/circleci.ts:677]:
```suggestion
              name: "--context",
```

Comment (fedeci) [reply src/circleci.ts:677]: Same for all the other options that are affected by the same problem. That 'string' should be added as an argument of the option

e.g.
```
{
	 name: "--context",
	 description: "Policy context for decision, default is 'config'",
	 dependsOn: ["policy_dir_path"],
   args: {
     name: "string"
   }
},
```

Comment (fedeci) [reply src/circleci.ts:677]: I have added a lint rule to prevent this issues from happening again :)

File: src/circleci.ts [hunk 3]
```diff
@@ -131,30 +368,333 @@ const completionSpec: Fig.Spec = {
...
+                "Enable/disable policy decision evaluation in build pipeline",
+              args: [
+                {
+                  name: "=true",
+                },
+                {
+                  name: "=false",
+                },
+              ],
+            },
```
Comment (fedeci) [inline src/circleci.ts:692]:
```suggestion
              name: "--enabled",
              requiresSeparator: true,
              description:
                "Enable/disable policy decision evaluation in build pipeline",
              args: {
              	name: "boolean",
              	isOptional: true,
              	suggestions: ["true", "false"],
              },
            },
```

**Review by fedeci — CHANGES_REQUESTED** [review]

**Comment by grant0417** [comment]
@fedeci looks like the linter is broken

**Comment by fedeci** [comment]
@grant not our fault https://github.com/amilajack/eslint-plugin-compat/issues/528

**Comment by jerdog** [comment]
You mean, lint failed lint? 🤣😭

**Comment by fedeci** [comment]
I mean, the lint for this PR is failing because of problems, but there is a dependency issue that is making all the other PRs and the master branch fail too. That one is not our fault :(

File: src/circleci.ts [hunk 4]
```diff
@@ -1,202 +1,686 @@
...
+          options: [
+            {
+              name: '--private',
+              description:
+                'Specify that this orb is for private use within your org, unlisted from the public registry',
+            },
+            {
+              name: '--sort',
+              description: 'Specify the sorting',
+              dependsOn: ['--namespace'],
```
Comment (fedeci) [inline src/circleci.ts:281]:
```suggestion
```

File: src/circleci.ts [hunk 5]
```diff
@@ -1,202 +1,686 @@
...
+          args: [
+            {
+              name: 'namespace/orb',
+              description: 'The namespace and orb to add to a category',
+            },
+            {
+              name: 'category name',
+              description:
+                'The name of the category to add the orb to, in quotes',
+              dependsOn: ['namespace/orb'],
```
Comment (fedeci) [inline src/circleci.ts:340]:
```suggestion
```

File: src/circleci.ts [hunk 6]
```diff
@@ -1,202 +1,686 @@
...
+        },
+        {
+          name: 'push',
+          description: 'Push policy bundle (or a single policy)',
+          args: {
+            {
+              name: 'policy_dir_path',
+              description: 'Policy file or directory path',
+            },
+          },
```
Comment (fedeci) [inline src/circleci.ts:580]:
```suggestion
          args: {
            name: 'policy_dir_path',
            description: 'Policy file or directory path',
          },
```

**Review by fedeci — APPROVED** [review]

**Pull request merged**